nsv13/code/modules/overmap/overmap.dm

```diff
@@ -37,6 +37,7 @@
 	var/last_slowprocess = 0
 
 	var/main_overmap = FALSE //There can only be one of these per game! This denotes that this ship is the "hero ship" and what the players fly. This links it to all the station areas by default
+	var/main_miner = FALSE //There can only be one of these per game! This denotes that this ship is the mining ship.
```

Comment:
I'd prefer that we change this to a define:
#define IS_MINING_SHIP(X) istype(X, /obj/structure/overmap/nostromo 
Or equivalent (see stuff like ismob for implementation)

Reply:
Does this also need to be done to main_overmap?

Reply:
yep

Reply:
Done.